modules/vcf2maf/1.3/vcf2maf.smk

```diff
@@ -397,44 +450,44 @@ def get_normalize_input(wildcards, todo_only = False):
 
     # Do we need to run CrossMap on this? Check the genome version
     if new_genome_build in original_genome_builds:
-        # Source matches. CrossMap not necessary. No maf normalization needed. 
+        # Source matches. CrossMap not necessary. No maf normalization needed.
         maf = expand(
-            CFG["dirs"]["vcf2maf"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}.gnomad_filtered.{filter}.{maf}", 
-            **wildcards, 
+            CFG["dirs"]["vcf2maf"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}.gnomad_filtered.{filter}.{maf}",
+            **wildcards,
             genome_build = new_genome_build
         )
         todo = "symlink"
-    elif new_genome_version in original_genome_version: 
-        # Source matches. CrossMap not necessary. 
+    elif new_genome_version in original_genome_version:
+        # Source matches. CrossMap not necessary.
         # Get the list index of the matching item
         index = original_genome_version.index(new_genome_version)
         # Get the input maf file
         maf = expand(
-            CFG["dirs"]["vcf2maf"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}.gnomad_filtered.{filter}.{maf}", 
-            **wildcards, 
+            CFG["dirs"]["vcf2maf"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}.gnomad_filtered.{filter}.{maf}",
+            **wildcards,
             genome_build = original_genome_builds[index]
         )
-        # Check if the chr prefix status is the same e.g. for grch37 vs. hs37d5. 
-        # If yes, symlink. If no, normalize. 
-        if VCF2MAF_GENOME_PREFIX[new_genome_build] == VCF2MAF_GENOME_PREFIX[original_genome_builds[index]]: 
+        # Check if the chr prefix status is the same e.g. for grch37 vs. hs37d5.
+        # If yes, symlink. If no, normalize.
+        if VCF2MAF_GENOME_PREFIX[new_genome_build] == VCF2MAF_GENOME_PREFIX[original_genome_builds[index]]:
             todo = "symlink"
         else:
             todo = "normalize"
     else:
         # Source doesn't match. CrossMap and normalization necessary.
-        target_build = "hg38" if new_genome_version == "grch38" else "hg19" 
+        target_build = "hg38" if new_genome_version == "grch38" else "hg19"
         maf = expand(
-            rules._vcf2maf_crossmap.output.maf, 
-            target_build = target_build, 
+            rules._vcf2maf_reannotate.output.maf,
```

Comment:
Meaningful change here: instead of the output of crossmap rule, use the outputs after re-annotation